app/library/Console/AbstractTask.php

```diff
@@ -140,5 +189,19 @@ protected function setUp()
         $this->output->setFormatter(new Line('%message%'));
 
         $this->basePath = dirname(app_path());
+
+        $this->climate = new CLImate();
+    }
+
+    /**
+     * @return bool
+     */
+    protected function assertNotWindows()
```

Comment:
1. assert* function should throw exception, not just return true/false
2. it is not a good place to introduce this function